### PR TITLE
Add residual + layernorm after attention

### PR DESCRIPTION
**Pull request by vliu15**

### CODE REVIEW

File: transformer/Layers.py [hunk 1]
```diff
@@ -34,10 +34,12 @@ def __init__(self, d_hidden, d_model, dropout=0.1):
         else:
             self.attn = DotProductAttention(d_model, d_hidden)
         self.dropout = nn.Dropout(dropout)
+        self.layernorm = nn.LayerNorm(d_model)
 
     def forward(self, enc_output, ses_hidden):
-        attn_distr = self.attn(enc_output, ses_hidden)
-        attn_output = self.dropout(attn_distr * enc_output)
+        attn_output = self.attn(enc_output, ses_hidden)
+        attn_output = self.dropout(attn_output * enc_output)
```
Comment (ademiadeniji) [inline transformer/Layers.py]: consider separating out the attention multiplication and dropout layer for clarity

**Review by ademiadeniji — APPROVED** [review]
